<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContactDialog</class>
 <widget class="QDialog" name="AddContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>420</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="personalDetailsGroup">
     <property name="title">
      <string>Personal Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="firstNameLabel">
        <property name="text">
         <string>First Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="3">
       <widget class="QLineEdit" name="firstNameEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lastNameLabel">
        <property name="text">
         <string>Last Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QLineEdit" name="lastNameEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QLineEdit" name="emailEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="homePHLabel">
        <property name="text">
         <string>Home PH:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="homePHEdit"/>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="mobileLabel">
        <property name="text">
         <string>Mobile:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="mobileEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="officePHLabel">
        <property name="text">
         <string>Office PH:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLineEdit" name="officePHEdit"/>
      </item>
      <item row="4" column="3">
       <widget class="QLabel" name="faxLabel">
        <property name="text">
         <string>FAX:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QLineEdit" name="faxEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="abnLabel">
        <property name="text">
         <string>ABN:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLineEdit" name="abnEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="addressGroup">
     <property name="title">
      <string>Address Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="addAddressButton">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="editAddressButton">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="removeAddressButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>11</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="4">
       <widget class="QTableView" name="addressList"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>firstNameEdit</tabstop>
  <tabstop>lastNameEdit</tabstop>
  <tabstop>emailEdit</tabstop>
  <tabstop>homePHEdit</tabstop>
  <tabstop>mobileEdit</tabstop>
  <tabstop>officePHEdit</tabstop>
  <tabstop>faxEdit</tabstop>
  <tabstop>abnEdit</tabstop>
  <tabstop>addressList</tabstop>
  <tabstop>addAddressButton</tabstop>
  <tabstop>editAddressButton</tabstop>
  <tabstop>removeAddressButton</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddContactDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddContactDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
